web/packages/test/config/beefy-relay.json

```diff
@@ -5,7 +5,7 @@
     },
     "beefy-activation-block": 0,
     "fast-forward-depth": 20,
-    "update-period": 0
+    "update-period": 20
```

Comment:
With this config the previous beefy relay degrades into only syncing handover updates.